typespec-extension/package.json

```diff
@@ -72,7 +73,7 @@
     "@typescript-eslint/parser": "~6.13.2",
     "@typescript-eslint/eslint-plugin": "~6.13.2",
     "eslint-plugin-deprecation": "~2.0.0",
-    "@azure-tools/typespec-azure-core": "0.37.0",
+    "@azure-tools/typespec-azure-core": "0.37.2",
```

Comment:
need to set to 0.37.2 to work, haven't digged into the cause yet..

Reply:
when you are about ready, you need to run `npm install` to update the package-lock file.

Reply:
added